### PR TITLE
Small formatting fixes concrete groups

### PR DESCRIPTION
**Pull request by fredrik-bakke**
Some small formatting fixes I did to concrete groups while reading through the files yesterday.

### CODE REVIEW

File: src/group-theory/concrete-groups.lagda.md [hunk 1]
```diff
@@ -101,91 +105,134 @@ module _
...
+```agda
+module _
+  {l : Level} (G : Concrete-Group l)
+  where
+
+  op-semigroup-Concrete-Group : Semigroup l
+  pr1 op-semigroup-Concrete-Group = set-Concrete-Group G
+  pr1 (pr2 op-semigroup-Concrete-Group) = mul-Concrete-Group' G
+  pr2 (pr2 op-semigroup-Concrete-Group) x y z =
+    inv (associative-mul-Concrete-Group G z y x)
```
Comment (EgbertRijke) [inline src/group-theory/concrete-groups.lagda.md:212]: This definition should really be `op-semigroup-Concrete-Group = op-Semigroup (semigroup-Concrete-Group G)` where `op-Semigroup` would be defined in a file `opposite-semigroups`.

File: src/group-theory/concrete-groups.lagda.md [hunk 2]
```diff
@@ -101,91 +105,134 @@ module _
...
+
+  is-unital-op-semigroup-Concrete-Group :
+    is-unital-Semigroup op-semigroup-Concrete-Group
+  pr1 is-unital-op-semigroup-Concrete-Group = unit-Concrete-Group G
+  pr1 (pr2 is-unital-op-semigroup-Concrete-Group) =
+    right-unit-law-mul-Concrete-Group G
+  pr2 (pr2 is-unital-op-semigroup-Concrete-Group) =
+    left-unit-law-mul-Concrete-Group G
```
Comment (EgbertRijke) [inline src/group-theory/concrete-groups.lagda.md:220]: This definition should be `is-unital-op-Group (group-Concrete-Group G)`

File: src/group-theory/concrete-groups.lagda.md [hunk 3]
```diff
@@ -101,91 +105,134 @@ module _
...
+
+  is-abstract-group-op-Concrete-Group' :
```
Comment (EgbertRijke) [inline src/group-theory/concrete-groups.lagda.md:222]: I don't think there is a predicate `is-abstract-group`, so we shouldn't be using it here in the name either

File: src/group-theory/torsors.lagda.md [hunk 4]
```diff
@@ -66,7 +67,11 @@ module _
   is-prop-is-torsor-Abstract-Group : is-prop is-torsor-Abstract-Group
   is-prop-is-torsor-Abstract-Group =
     is-prop-type-Prop is-torsor-Abstract-Group-Prop
+```
+
+### The type of torsors
```
Comment (EgbertRijke) [inline src/group-theory/torsors.lagda.md:72]: Nice!

File: src/group-theory/torsors.lagda.md [hunk 5]
```diff
@@ -591,8 +606,7 @@ module _
     (X : Torsor-Abstract-Group G l1) →
     Id (principal-Torsor-Abstract-Group G) X ≃ Eq-Torsor-Abstract-Group X
   equiv-Eq-equiv-Torsor-Abstract-Group X =
-    ( pair
-      ( Eq-equiv-Torsor-Abstract-Group X)
+    ( ( Eq-equiv-Torsor-Abstract-Group X) ,
       ( is-equiv-Eq-equiv-Torsor-Abstract-Group X)) ∘e
```
[the remark this answers is not in the document]
Comment (fredrik-bakke) [reply src/group-theory/torsors.lagda.md:610]: I wrote an issue to bring attention to this.

File: src/group-theory/concrete-groups.lagda.md [hunk 6]
```diff
@@ -101,91 +107,117 @@ module _
...
-  pr1 (pr2 (pr2 op-abstract-group-Concrete-Group)) = inv-Concrete-Group
-  pr1 (pr2 (pr2 (pr2 op-abstract-group-Concrete-Group))) =
-    right-inverse-law-mul-Concrete-Group
-  pr2 (pr2 (pr2 (pr2 op-abstract-group-Concrete-Group))) =
-    left-inverse-law-mul-Concrete-Group
+  is-group-Concrete-Group : is-group semigroup-Concrete-Group
+  pr1 is-group-Concrete-Group = is-unital-semigroup-Concrete-Group
+  pr2 is-group-Concrete-Group = is-group-Concrete-Group'
+
+  group-Concrete-Group : Group l
```
Comment (EgbertRijke) [inline src/group-theory/concrete-groups.lagda.md:198]: Good call to name this `group-Conctrete-Group`

File: src/group-theory/torsors.lagda.md [hunk 7]
```diff
@@ -66,7 +67,11 @@ module _
   is-prop-is-torsor-Abstract-Group : is-prop is-torsor-Abstract-Group
```
Comment (EgbertRijke) [inline src/group-theory/torsors.lagda.md:67]: in a future pull request, we can also rename `Abstract-Group` to `Group`, but this PR looks good already.

**Pull request merged**